test/common/distributed_helper.erl

```diff
@@ -20,6 +20,18 @@ is_sm_distributed() ->
 is_sm_backend_distributed(ejabberd_sm_mnesia) -> true;
 is_sm_backend_distributed(Other) -> {false, Other}.
 
+db_engine() ->
+    escalus_ejabberd:rpc(mongoose_rdbms, db_engine, [domain_helper:host_type()]).
```

Comment:
I meant, we don't need `escalus_ejabberd` here. We can call `distributed_helper:rpc(distributed_helper:mim(), mongoose_rdbms, db_engine, [domain_helper:host_type()])`, we have done this a million time in tests. The point is about removing all usages of `escalus_ejabberd`.